src/ralph/schemas/edx/converters/xapi/constants.py

```diff
@@ -0,0 +1,30 @@
+"""Constants for xAPI specifications"""
+
+# Languages
+EN = "en"
+
+# xAPI activities
+XAPI_ACTIVITY_PAGE = "http://activitystrea.ms/schema/1.0/page"
+
+# xAPI verbs
+XAPI_VERB_VIEWED = "http://id.tincanapi.com/verb/viewed"
+
+# xAPI extensions
+XAPI_EXTENSION_ACCEPT_LANGUAGE = "https://www.edx.org/extension/accept_language"
+XAPI_EXTENSION_AGENT = "https://www.edx.org/extension/agent"
+XAPI_EXTENSION_COURSE_ID = "https://www.edx.org/extension/course_id"
+XAPI_EXTENSION_COURSE_USER_TAGS = "https://www.edx.org/extension/course_user_tags"
+XAPI_EXTENSION_HOST = "https://www.edx.org/extension/host"
+XAPI_EXTENSION_IP = "https://www.edx.org/extension/ip"
+XAPI_EXTENSION_ORG_ID = "https://www.edx.org/extension/org_id"
+XAPI_EXTENSION_PATH = "https://www.edx.org/extension/path"
+XAPI_EXTENSION_REFERER = "https://www.edx.org/extension/referer"
+XAPI_EXTENSION_REQUEST = "https://www.edx.org/extension/request"
+XAPI_EXTENSION_USER_ID = "https://www.edx.org/extension/user_id"
```

Comment:
```suggestion
XAPI_EXTENSION_AGENT = "http://id.tincanapi.com/extension/browser-info"
XAPI_EXTENSION_COURSE_ID = "https://www.edx.org/extension/course_id"
XAPI_EXTENSION_COURSE_USER_TAGS = "https://www.edx.org/extension/course_user_tags"
XAPI_EXTENSION_HOST = "https://www.edx.org/extension/host"
XAPI_EXTENSION_IP = "http://id.tincanapi.com/extension/ip-address"
XAPI_EXTENSION_ORG_ID = "https://www.edx.org/extension/org_id"
XAPI_EXTENSION_PATH = "https://www.edx.org/extension/path"
```
WDYT ?
If we remove the referer because of sensitive information may be we should remove the client request arguments too (GET parameters / POST data)
There are extensions in the [tincanapi extentsions page](https://registry.tincanapi.com/#home/extensions) that match some of ours - should we use those instead? (ip / agent)